### ISSUE TITLE
Setting SelectedSegment to -1 to clear selected segment causes null reference in Android

### ISSUE THREAD

**Issue by TrueMoniker** [yurkinh/Plugin.Maui.SegmentedControl#4]
Setting SelectedSegment to -1 to clear selected segment causes null reference in Android.
I'd expect the -1 to have no segment selected. If -1 is not supported, how do I deselect a value in the Segmented Control?

** Reproducable Example **
Open a view on Android with the SegmentedControl below:
```xaml
<segmented:SegmentedControl SelectedSegment="-1">
    <segmented:SegmentedControl.Children>
        <segmented:SegmentedControlOption Text="yes" />
        <segmented:SegmentedControlOption Text="no" />
    </segmented:SegmentedControl.Children>
</segmented:SegmentedControl>
```

** Error Message **
`System.NullReferenceException: Object reference not set to an instance of an object.
  at at Plugin.Maui.SegmentedControl.Handlers.SegmentedControlHandler.MapSelectedTextColor(SegmentedControlHandler handler, SegmentedControl control)`

**Comment by yurkinh** [comment]
Hello @TrueMoniker 
Thanks for reporting an issue.
I will take a look at it.

**Comment by yurkinh** [comment]
Hello @TrueMoniker 
Check please the latest v1.0.8 from nuggets - it contains a fix for this issue from pull #5

**Comment by TrueMoniker** [comment]
Hello @yurkinh @MykhailoDav 
v1.0.8 fixed it. Thank you so much! Very Excellent
